charts/eks-operator/templates/deployment.yaml

```diff
@@ -27,7 +30,11 @@ spec:
           value: {{ .Values.noProxy }}
 {{- if .Values.additionalTrustedCAs }}
         volumeMounts:
-          - mountPath: /etc/ssl/certs/ca-additional.pem
+          - mountPath: /etc/rancher/ssl/ca-additional.pem
+            name: tls-ca-additional-volume
+            subPath: ca-additional.pem
+            readOnly: true
+          - mountPath: /etc/pki/trust/anchors/ca-additional.pem
```

Comment:
Is this meant to only mount `ca-additional.pem`, or did you want to mount the whole anchors directory?

Why not `/etc/ssl/certs`? I don't see any reference to `ca-additional.pem` being anywhere but `/etc/rancher/ssl` or `/etc/ssl/certs`

Reply:
We only want to mount the ca-additional.pem file.

The reason for the directory is here: https://documentation.suse.com/suse-caasp/4.5/html/caasp-admin/_troubleshooting_3.html#_error_x509_certificate_signed_by_unknown_authority